ui/src/app/views/invitation-code/component.css

```diff
@@ -0,0 +1,20 @@
+.input {
+  width: 100%;
+}
+
+.error {
+  background-color: #ffeeee;
+  padding: 0 0.5rem;
+  margin-top: 0.2rem;
+}
+
+.label{
+  width:10px;
```

Comment:
Is there a specific reason to use `px` over `rem` here?

Reply:
No particular reason...since was not sure about the styling i used px....will change it to rem